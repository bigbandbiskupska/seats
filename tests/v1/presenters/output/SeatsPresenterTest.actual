[
	'id' => 301,
	'schema_id' => 1,
	'state' => 1,
	'price' => 250,
	'x' => 100,
	'y' => 300,
	'row' => 40,
	'col' => 30,
]
