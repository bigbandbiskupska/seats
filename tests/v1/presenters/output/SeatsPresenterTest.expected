[
	'x' => 100,
	'y' => 300,
	'row' => 40,
	'col' => 30,
	'schema_id' => 1,
	'price' => 250,
	'id' => 301,
]
